### PR TITLE
Update from my repo

### PR DESCRIPTION
**Pull request by exp111**
So this is a huge one and is probably really messy but here are the changes:
*Auto Knife & Auto Zeus (own file but can probably be moved to Triggerbot file; in Triggerbot tab; uses the same key as triggerbot right now)
*Custom Watermarks (default is Antario; Color can be changed in color window; in misc tab)
*Bomb Timer in the top left corner (Same color as watermark; places itself automatically under watermark if activated; misc tab too)

### CODE REVIEW

**Comment by exp111** [comment]
Stuck on searching the compiler? Had that problem when I switched to antario too. 
Thought it was a problem with g++, so I did a full system update and it worked. 

Maybe try restarting your pc or reinstalling cmake/gcc. 
Don't know why it happens, maybe does @Wando1423 know?
I'll test a bit around when I get home but it worked fine for me

File: src/Hacks/autoknife.cpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+	if (itemDefinitionIndex == ItemDefinitionIndex::WEAPON_TASER && activeWeapon->GetAmmo() == 0)
+		return;
+
+	Vector traceStart, traceEnd;
+	trace_t tr;
+
+	QAngle viewAngles;
+	engine->GetViewAngles(viewAngles);
+	QAngle viewAngles_rcs = viewAngles + *localplayer->GetAimPunchAngle() * 2.0f;
```
Comment (Wando1423) [inline src/Hacks/autoknife.cpp]: Code style, no underscores in variable names.

File: make [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```
Comment (Wando1423) [inline make]: I wouldnt add that. This one can be the reason of compilation not working.
I'd rather use "compile" name.

File: src/Hacks/autoknife.cpp (hunk 1 above)
Comment (exp111) [inline src/Hacks/autoknife.cpp]: I just pasted the Triggerbot ray check…I'll rework the whole autoknife and integrate it into the triggerbot later

**Comment by Wando1423** [comment]
Ok, for now its merged. If you'd make some changes to it later, let me know.

**Comment by exp111** [comment]
Thanks for reviewing. Making some changes right now. I'll put a request if I'm finished

**Pull request merged**